tools/metrics/histograms/metadata/cobalt/histograms.xml

```diff
@@ -55,6 +70,32 @@ Always run the pretty print utility on this file after editing:
   </summary>
 </histogram>
 
+<histogram name="Media.PipelineStatus.AudioVideo.{VideoCodec}.{Pipeline}"
+    enum="PipelineStatus" expires_after="never">
+<!-- expires-never: Media pipeline health metric. -->
```

Comment:
The "expires-never" tag is an indication/reasoning why you don't want it to expire. Something like "Needed for longitudinal analysis" or something like that would be better.